benches/osrank_naive.rs

```diff
@@ -0,0 +1,164 @@
+#![macro_use]
+extern crate criterion;
+extern crate rand;
+extern crate rand_xorshift;
+
+use criterion::{Criterion, Benchmark};
+use criterion::criterion_group;
+use criterion::criterion_main;
+use osrank::protocol_traits::graph::{Graph, GraphObject};
+use osrank::protocol_traits::ledger::{MockLedger, LedgerView};
+use osrank::types::{Artifact, ArtifactType, Dependency, DependencyType, Network, Osrank, Weight};
+use osrank::algorithm::osrank_naive;
```

Comment:
I am surprised clippy/cargo didn't warn you about "merging" those imports into:

```
use osrank::algorithm::{osrank_naive, random_walk, rank_network};
```

Perhaps it's time for your editor to be properly set up or it's just a cargo quirk where bench stuff is not linted? 😉

Reply:
Yes, I guess it is time 🤦‍♂